content/blog/tables-are-hard-2/index.md

```diff
@@ -19,6 +19,11 @@ This tutorial is aimed at relative beginners to web development, and will keep t
 
 [Final Code](https://github.com/pixie-io/pixie-demos/tree/main/react-table) | [Live Demo](https://pixie-io.github.io/pixie-demos/react-table)
 
+_This article is part of a series_:
```

Comment:
I'd move this below the video on the next line.

Reply:
Good catch, done so it's the same between both articles.